static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -230,6 +231,34 @@ export function StacktraceLink({frame, event, line, disableSetup}: StacktraceLin
     refetch();
   };
 
+  const filePath =
+    frame.filename && frame.lineNo ? frame.filename + ':' + frame.lineNo : '';
+
+  const {onClick: handleCopyPath} = useCopyToClipboard({
+    text: filePath,
+    successMessage: t('File path copied to clipboard'),
+    errorMessage: t('Failed to copy file path'),
+    onCopy: () => {
+      trackAnalytics('integrations.stacktrace_file_path_copied', {
+        view: 'stacktrace_issue_details',
```

Comment:
This isn't necessarily going to always be on the issue details page, it could be rendered in other parts of the app as well (not sure if it actually is, but if it were this would be wrong)